usrsctplib/netinet/sctputil.c

```diff
@@ -6704,10 +6704,10 @@ sctp_sorecvmsg(struct socket *so,
 					if (SCTP_BASE_SYSCTL(sctp_logging_level) & SCTP_SB_LOGGING_ENABLE) {
 						sctp_sblog(&so->so_rcv, control->do_not_ref_stcb?NULL:stcb, SCTP_LOG_SBFREE, (int)cp_len);
 					}
-					atomic_subtract_int(&so->so_rcv.sb_cc, cp_len);
+					atomic_subtract_int(&so->so_rcv.sb_cc, (int32_t)cp_len);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/078bd24ac1768bc6dc87d40b030434cba1987ea3.